Userland/Applications/HexEditor/HexEditorWidget.cpp

```diff
@@ -209,6 +223,13 @@ HexEditorWidget::HexEditorWidget()
         set_search_results_visible(action.is_checked());
     });
 
+    m_selection_information_format_mode_action = GUI::Action::create_checkable("Selection info in hex", [&, update_statusbar_selection_and_offset_info](auto& action) {
```

Comment:
This is not a good name. UI text follows book case and usually is phrased as a command.
```suggestion
    m_selection_information_format_mode_action = GUI::Action::create_checkable("Show Selection Info in Hex", [&, update_statusbar_selection_and_offset_info](auto& action) {
```